crawl-ref/source/monster.cc

```diff
@@ -6347,6 +6347,14 @@ bool monster::is_web_immune() const
             || mons_genus(type) == MONS_MOTH
```

Comment:
The comment at the top of this function needs to be updated. (Probably just deleting the 'currently' section.)